src/serviceClients/jobManagerWrapper.ts

```diff
@@ -49,6 +50,26 @@ export class JobManagerWrapper extends JobManagerClient {
     }
   }
 
+  public async createInitUpdateJob(
+    id: string,
+    version: string,
+    internalId: string,
+    data: UpdateRasterLayer,
+    jobAction: JobAction
+  ): Promise<ICreateJobResponse> {
+    const logCtx: LogContext = { ...this.logContext, function: this.createUpdateJob.name };
+    const taskParams: ITaskParameters[] = [{ blockDuplication: true }];
+    try {
+      const jobType = jobAction === JobAction.UPDATE ? this.ingestionUpdateJobType : this.ingestionSwapUpdateJobType;
```

Comment:
please move the "JobAction " logic outside of this method and get the jobType as a parmeter of this method

Reply:
why? this is an internal logic with params that are defined in the contractor. it creates more junk and mess when creating to many "small" functions of one line logic.

Reply:
but you previously asked for this inside the Ctor:
 this.ingestionUpdateJobType = config.get<string>('jobManager.ingestionUpdateJobType');
  this.ingestionSwapUpdateJobType = config.get<string>('jobManager.ingestionSwapUpdateJobType');
    
 why woud you want the ingestionManager to hold them?

Reply:
done - moved logic to ingestionManager and removed enums